server/src/internalClusterTest/java/org/elasticsearch/snapshots/GetSnapshotsIT.java

```diff
@@ -1000,4 +1020,97 @@ public void testAllFeatures() {
 
         assertEquals(0, remaining);
     }
+
+    private static Set<SnapshotId> removeRandomSnapshotDetails(List<String> repositories) {
```

Comment:
Might better to name it removeSnapshotDetailsFromRandomSnapshots, or something. Right not it suggests to me that random details will be removed -- sometimes this field, sometimes that other field. But a comment will clarify 🤷‍♀️